target/dita/regions/regions.dita

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+&lt;!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd"&gt;
+<topic id="links_1">
+ <title>
+  Regions
+ </title>
+ <body>
+  <imagemap>
+   <image href="./Content/Images/world-map.gif"/>
```

Comment:
the parser needs to include the step to copy this source image file into `target/dita/regions/Content/Images`

Reply:
ok, what about this step https://github.com/DeepBlueCLtd/LegacyMan/blob/96336c5408c30edde35010c0a1a69c8c481c94f0/parser/parser.py#L39

Reply:
Aah, it could be case sensitive.  Let's keep it all lower-case.